client/apps/edit/components/section_image_set/index.coffee

```diff
@@ -0,0 +1,263 @@
+#
+# Image Set section that allows uploading a mix of images and artworks
+#
+
+# Using `try` here b/c Scribe is an AMD module that doesn't play nice when
+# requiring it for testing in node.
+try
+  Scribe = require 'scribe-editor'
+  scribePluginToolbar = require 'scribe-plugin-toolbar'
+  scribePluginSanitizer = require '../../lib/sanitizer.coffee'
+  scribePluginLinkTooltip = require 'scribe-plugin-enhanced-link-tooltip'
+_ = require 'underscore'
+gemup = require 'gemup'
+React = require 'react'
+toggleScribePlaceholder = require '../../lib/toggle_scribe_placeholder.coffee'
+sd = require('sharify').data
+icons = -> require('./icons.jade') arguments...
+Autocomplete = require '../../../../components/autocomplete/index.coffee'
+Artwork = require '../../../../models/artwork.coffee'
+{ div, section, h1, h2, span, img, header, input, nav, a, button, p, ul, li, strong } = React.DOM
+{ crop, resize, fill } = embedly = require('embedly-view-helpers')(sd.EMBEDLY_KEY)
+
+module.exports = React.createClass
```

Comment:
:clap: Nice work on this component! It's a little long in line numbers, but not _too_ long yet. If this grows in size I'd encourage trying to find way to break it down into even smaller parts.